app/services/search/query_builders/user.rb

```diff
@@ -21,8 +21,8 @@ def initialize(params:)
 
         @params = params.deep_symbolize_keys
 
-        # default to excluding users who are banned
-        @params[:exclude_roles] = ["banned"]
+        # default to excluding users who are suspended
+        @params[:exclude_roles] = ["suspended"]
```

Comment:
Same thing. If "legacy" users still have the `banned` role, this will show them in the search. It's probably un unlikely scenario BUT if the data update script fails or if for whatever reason takes too long, we might show banned users in the search.

Reply:
We can leave them, then I will follow up with a PR to remove them down the road. Safety first :safety_pin: